spec/query_spec.rb

```diff
@@ -447,20 +451,14 @@ def test_method
 
   describe '#distinct' do
     before do
-      over_z = client.query do
-        create ref('indexes'), name: 'query_over_z', source: @test_class, values: [{ path: 'data.z' }]
-      end
-      wait_for_index(over_z[:ref])
-      @test_over_z = over_z[:ref]
-
       @refs = []
       @refs << client.query { create @test_class, data: { z: 0 } }[:ref]
       @refs << client.query { create @test_class, data: { z: 1 } }[:ref]
       @refs << client.query { create @test_class, data: { z: 1 } }[:ref]
     end
 
     it 'performs distinct' do
-      set = Fauna::Query.match(@test_over_z)
+      set = Fauna::Query.match(@test_by_z)
       distinct = Fauna::Query.distinct(set)
 
       expect(get_set_data(set)).to eq([0, 1, 1])
```

Comment:
I was trying to see if I could get the test to pass without cloud running its mappers (they're off while the migration happens). That doesn't seem to be happening, though. So I've put this off until the migration is complete.